### DISCUSSION TITLE
Can I host this on free or shared webhosting?

### DISCUSSION THREAD

**Discussion by ririko5834** [shlinkio/discussions#40] [Help wanted] [answered]
Hello, can I host this on free webhosting, without shell console?

**Comment by acelaya** [comment, marked as answer]
It's not impossible, but it's not Shlink's primary goal, so it's not officially supported and you will have to do some manual hacks.

Take a look at this to see if it helps https://shlink.io/documentation/troubleshooting/#how-to-install-shlink-on-a-shared-hosting
